components/LoginModal.vue

```diff
@@ -472,46 +472,50 @@ export default {
             })
         }
       } else if (this.email && this.password) {
-        // Login
-        this.authStore
-          .login({
-            email: this.email,
-            password: this.password,
-          })
-          .then(() => {
-            // We are now logged in. Prompt the browser to remember the credentials.
-            if (window.PasswordCredential) {
-              try {
-                // We used to pass in the DOM element, but in Chrome 92 that causes a crash.
-                const c = new window.PasswordCredential({
-                  id: this.email,
-                  password: this.password,
-                })
-                navigator.credentials
-                  .store(c)
-                  .then(function () {
-                    self.pleaseShowModal = false
-                  })
-                  .catch((err) => {
-                    console.error('Failed to save credentials', err)
+        if (this.emailError || this.passwordError) {
+          this.nativeLoginError = 'Please fill out the form.'
```

Comment:
Is this the right error message?  We're inside `if (this.email && this.password)` - so we don't get this message in the case where the form hasn't been filled in - we just get the red borders:

![image](https://github.com/Freegle/iznik-nuxt3/assets/56696/14e295d1-13d1-425e-b2fa-5f945d74461c)

Should we just lose the `if (this.email && this.password` check?

Reply:
Is there any specific reason to send an API request with an invalid email/password? I've just added this check to avoid unnecessary calls to the API.

Reply:
There's no reason to send an invalid API call.  But when I tested the code, if you didn't fill in any forms and just hit the log in button then you get the red borders with no error message.  We need to show the error message.

Reply:
@edwh it seems that it's the same on production 
<img width="1250" alt="image" src="https://github.com/Freegle/iznik-nuxt3/assets/11361743/ff904c65-5a6c-4c30-b1cf-d3b5d614fb83">
I'll prepare a fix

Reply:
@edwh Done. https://github.com/Freegle/iznik-nuxt3/pull/50/commits/5e1c719c08fd33718e872acd8174c5bb20826368